tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs

```diff
@@ -279,23 +279,28 @@ private unsafe static void PrimitiveParserByteSpanToUInt32(string text)
             }
         }
 
-        [Benchmark(Skip = "The generic type 'System.Collections.Generic.List`1' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
-        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
+        [Benchmark]
+        [InlineData(0)]
```

Comment:
But you can create an array of byte arrays, then implicit cast to span kicks in during the perf loop.

Does somethign like this not work:
```csharp
        [Benchmark]
        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
        {
            byte[][] byteSpanList = (byte[][])Array.CreateInstance(typeof(byte[]), s_UInt32TextArray.Length);
            for (var i = 0; i < s_UInt32TextArray.Length; i++)
            {
                byteSpanList[i] = Encoding.UTF8.GetBytes(s_UInt32TextArray[i]);
            }

            foreach (var iteration in Benchmark.Iterations)
            {
                using (iteration.StartMeasurement())
                {
                    for (int i = 0; i < LoadIterations; i++)
                    {
                        ReadOnlySpan<byte> utf8ByteSpan = byteSpanList[i % 10];
                        uint value;
                        PrimitiveParser.InvariantUtf8.TryParseUInt32(utf8ByteSpan, out value);
                        DoNotIgnore(value, 0);
                    }
                }
            }
        }
```